jobs/multi-arch-pipeline.Jenkinsfile

```diff
@@ -480,6 +480,16 @@ EOF
                     string(name: 'FCOS_CONFIG_COMMIT', value: params.FCOS_CONFIG_COMMIT)
                 ]
             }
+            stage('Kola:Kubernetes') {
+                // We consider the Kubernetes kola tests to be a followup job, so we use `wait: false` here.
+                build job: 'kola-kubernetes', wait: false, parameters: [
+                    string(name: 'STREAM', value: params.STREAM),
+                    string(name: 'VERSION', value: newBuildID),
+                    string(name: 'S3_STREAM_DIR', value: s3_stream_dir),
+                    string(name: 'ARCH', value: basearch),
+                    string(name: 'FCOS_CONFIG_COMMIT', value: params.FCOS_CONFIG_COMMIT)
+                ]
+            }
```

Comment:
oh nice.. Does the test work against `aarch64` ?

Reply:
I'm not sure, but I figured we might as well find out and follow-up if it doesn't. The cri-o installer script at least accounts for it: https://github.com/cri-o/cri-o/blob/367232bb1bc7ab3a3bd5a863daf701b036966473/scripts/get#L43-L54